scripts/generate-hybrid-regions.py

```
@@ -883,84 +907,84 @@ def save_regions(self, filename: str = "../lib/locations/regions_comprehensive.j
                 "regions": self.regions
             }
 
             with open(filename, 'w', encoding='utf-8') as f:
                 json.dump(output_data, f, indent=2, ensure_ascii=False)
 
-            self.log(f"💾 Saved {len(self.regions)} regions to {filename}", "SUCCESS")
-
+            self.log(f"💾 Saved {len(self.regions)} regions!", "SUCCESS")
             self.generate_statistics()
 
         except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)